Odata-docs/webapi-8/entity-routing.md

```diff
@@ -0,0 +1,508 @@
+---
+title:  "Entity Routing in ASP.NET Core OData 8"
+description: Entity Routing in ASP.NET Core OData 8.
+date:   2022-12-07
+ms.date: 12/7/2022
+author: gathogojr
+ms.author: jogathog
+---
+
+# Entity Routing in ASP.NET Core OData 8
+**Applies To**:[!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v8.md)]
+
+This tutorial shows how ASP.NET Core OData 8 supports entity routing. A basic understanding of routing in ASP.NET Core OData 8 is assumed. If you're unfamiliar with routing in ASP.NET Core OData 8, you may want to go through the [routing overview](/odata/webapi-8/routing-overview) tutorial.
+
+## Prerequisites
+
+[!INCLUDE[](../includes/appliesto-webapi-v8-net-prereqs-vs.md)]
+
+## Introduction
+OData entity routing convention supports the following route templates:
+
+| Request Method | Route Template |
+|----------------|----------------|
+| GET | ~/{entityset}/{key} |
+| GET | ~/{entityset}/{key}/{cast} |
+| PUT | ~/{entityset}/{key} |
+| PUT | ~/{entityset}/{key}/{cast} |
+| PATCH | ~/{entityset}/{key} |
+| PATCH | ~/{entityset}/{key}/{cast} |
+| DELETE | ~/{entityset}/{key} |
+| DELETE | ~/{entityset}/{key}/{cast} |
+
+**Note:** OData routing supports canonical parentheses-style key in addition to key-as-segment. Currently, ASP.NET Core OData 8 does not support key-as-segment convention in multi-part keys scenarios.
+
+To illustrate entity routing convention, let's build a contrived OData service.
```

Comment:
is there a simpler word that can be used in place of contrived in all the pages?

Reply:
@ElizabethOkerio I can use sample :)